docs/docs/how-tos/nebari-gcp.md

```diff
@@ -4,20 +4,20 @@ title: Deploy Nebari on GCP
 description: A basic overview of how to deploy Nebari on GCP.
 ---
 
-## Introduction
-
-This guide is to help first-time users set up a Google Cloud Platform account specifically for the purpose of using and deploying Nebari at a production scale. In this guide we
-will walk you through the following steps:
-
```

Comment:
Add a note commenting that from previous versions, the user might required to install gcloud since this is now not needed. (something around this)

Reply:
Great point, thanks! I added a note